app/models/scheme.rb

```diff
@@ -4,7 +4,8 @@ class Scheme < ApplicationRecord
 
   scope :search_by_code, ->(code) { where("code ILIKE ?", "%#{code}%") }
   scope :search_by_service_name, ->(name) { where("service_name ILIKE ?", "%#{name}%") }
-  scope :search_by, ->(param) { search_by_code(param).or(search_by_service_name(param)) }
+  scope :search_by_postcode, ->(postcode) { joins(:locations).where("REPLACE(locations.postcode, ' ', '') ILIKE ?", "%#{postcode.delete(' ')}%") }
```

Comment:
I wonder if we might be better off removing the space in the data?

Reply:
We could store without any spaces and all capitals for example when creating/importing data, part of creating locations for schemes ticket ( it doesnt exist yet, but I will make it before end of the day ).

Reply:
Yup. Sounds good, might look into implementing it as part of this PR

Reply:
If we do (which would be good), we probably also need a rake task we can run to fix production data we already have to that standard (could just run it through console but probably good to start getting into the habit of doing it properly).

Reply:
Looks like all production data for logs already normalises postcode values so think we're good here.